jekyll/_cci2/runner-machine-migration.adoc

```diff
@@ -0,0 +1,147 @@
+
+The final step is to move the agent configuration file. The default location for this is `/etc/opt/circleci/launch-agent-config.yaml`. The file should be moved to `/etc/circleci-runner/circleci-runner-config.yaml` using the following command
+
+```shell
+mv /etc/opt/circleci/launch-agent-config.yaml /etc/circleci-runner/c sircleci-runner-config.yaml
+```
+
+Alternatively you can let the Machine Agent install process create a config file template and copy the existing config into the newly created file (This process is documented below)
+
+[#uninstalling-launch-agent-linux-se]
+=== RHEL/Rocky Linux
+
+In addition to the steps listed above, the SELinux policy should be uninstalled using the following command:
+
+```shell
+sudo semodule -r circleci_launch_agent
+```
+
+[#uninstalling-launch-agent-macos]
+=== MacOS
+
+If the agent has been installed as a launchd service, run the following commands to stop the service and remove the service file:
+
+```shell
+sudo launchctl unload '/Library/LaunchDaemons/com.circleci.runner.plist'
+
+sudo rm `/Library/LaunchDaemons/com.circleci.runner.plist`
+```
+
+Now the Launch Agent binary can be removed from the machine. The default install location is `/opt/circleci/circleci-launch-agent` but this may differ on your machine if the agent was installed somewhere else. The entire `/opt/circleci` subdirectory can be removed at this point.
+
+The final step is to move the agent configuration file. The default location for this is `/Library/Preferences/com.circleci.runner/launch-agent-config.yaml`. The file should be moved to `$HOME/Library/Preferences/com.circleci.runner/config.yaml` 
+
+Alternatively you can let the Machine Agent install process create a config file template and copy the existing config into the newly created file (This process is documented below)
+
+[#uninstalling-launch-agent-windows]
+=== Windows
+
+To uninstall the agent you can use the uninstall script. The https://github.com/CircleCI-Public/runner-installation-files/blob/main/windows-install/Uninstall-CircleCIRunner.ps1[`Uninstall-CircleCIRunner.ps1` script] can be downloaded from GitHub to an easily accessible location, and then run with the following command (from an administrator powershell console):
+
+```shell
+./Uninstall-CircleCIRunner.ps1
+```
+
+This will remove all Launch Agent components including the configuration file. If you want to reuse the values with the new install, you should save the file to a temporary location while migrating. It is stored by default at `$env:ProgramFiles\CircleCI`
+
+
+[#uninstalling-machine-agent]
+== Installing the Machine Agent
+
+The new agent has a streamlined installation experience on Linux/MacOS thanks to new packaged options.
+
+[#uninstalling-machine-agent-linux]
+=== Linux
+
+At present deb and rpm packages are available for linux. Use the following commands to install them
+
+NOTE: These commands should be run with `bash`. The packagecloud script has issues when run with `zsh`
```

Comment:
If we remove the query parameter from the curl request, `curl -s https://packagecloud.io/install/repositories/circleci/runner/script.deb.sh` zsh doesn't mind the URL anymore.